async_firebase/base.py

```diff
@@ -166,7 +166,7 @@ async def send_request(
         :param content: request content
         :return: HTTP response
         """
-        async with httpx.AsyncClient(base_url=self.BASE_URL) as client:
+        async with httpx.AsyncClient(base_url=self.BASE_URL, timeout=None) as client:
```

Comment:
In my opinion, this is a rather exception than a common scenario, and thus I would recommend addressing this issue differently. As I see it, the client must be able to modify the request parameters, like timeout, keep-alive connections, etc.

Reply:
Yeah I agree. It would be better if the `AsyncFirebaseClient` can have configs for timeout, keep-alive, etc

Reply:
Ok, let me prioritize some work.

Reply:
Hey @jeonghanjoo,
The feature you are looking for is already available in version 3.2.0. Check it out
 https://github.com/healthjoy/async-firebase/releases/tag/v3.2.0
https://pypi.org/project/async-firebase/

Reply:
@akalex Sorry for late reply, Great! Thank you!